src/components/Forum/ItemDetail.module.css

```diff
@@ -52,25 +79,80 @@
 }
 
 .commentDiv {
+
     color: white;
     font-size: 1rem;
     outline: none;
     background-color: transparent;
     border-bottom: 1px solid black;
     padding: 15px 20px;
+    display: flex;
+    flex-direction: column;
    
 }
 
 .commentsTitle {
+
     border-bottom: 2px solid black;
     font-weight: bold;
     color: white;
 }
 
+.addComment>textarea {
+
+    height: 200px;
+    background: rgba(255, 255, 255, 0.2);
+    color:white;
+    font-weight: bolder;
+    font-size: 150%;
+    outline: none;
+    border: 0;
+    padding: 20px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    font-family: 'Rationale';
+    width: 100%;
+}
+
+
+
+.addComment textarea::-moz-placeholder{
+    color: white;
+}
+
+
+.addComment textarea::-webkit-input-placeholder {
+    color: white;
+  }
+  
+  .addComment textarea:-moz-placeholder { 
+    color: white; 
+  }
+  
+  .addComment textarea::-moz-placeholder {  
+    color: white; 
+  }
+  
+ .addComment  textarea:-ms-input-placeholder {
+    color: white;  
+  }
+  
+  .addComment textarea::placeholder {
+    color: white; 
+  }
```

Comment:
repeated line of code
.addComment ... color: white -> they could be settled in one line with commas